scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -416,10 +416,10 @@ private void releaseBusyNodesWithNoRunningTask(RMProxy rmProxy, RecoveredSchedul
         return busyNodesWithTask;
     }
 
-    /**
+    /*
```

Comment:
it will be not java doc any more?

Reply:
It was not java doc even before. I removed redundant asterix

Reply:
yes this is not javadoc